harness/determined/cli/user.py

```diff
@@ -161,7 +164,14 @@ def whoami(parsed_args: Namespace) -> None:
         Cmd("change-password", change_password, "change password for user", [
             Arg("target_user", nargs="?", default=None, help="name of user to change password of")
         ]),
-        Cmd("logout", log_out_user, "log out user", []),
+        Cmd("logout", log_out_user, "log out user", [
+            Arg(
+                "--all",
+                "-a",
+                action="store_true",
+                help="log all users out",
```

Comment:
can you specify:

```
log out of all cached sessions for the current master
```

?

It still isn't truly _all_ sessions, just all the ones for the current master.  `auth.json` tracks sessions for multiple masters.